dd-java-agent/src/main/java/datadog/trace/bootstrap/AgentBootstrap.java

```diff
@@ -46,6 +46,7 @@
 public final class AgentBootstrap {
   static final String LIB_INJECTION_ENABLED_ENV_VAR = "DD_INJECTION_ENABLED";
   static final String LIB_INJECTION_FORCE_SYS_PROP = "dd.inject.force";
+  static final String LIB_INSTRUMENTATION_SOURCE_SYS_PROP = "instrumentation.source";
```

Comment:
```suggestion
  static final String LIB_INSTRUMENTATION_SOURCE_SYS_PROP = "dd.instrumentation.source";
```
need to include the `dd.` prefix since we're using this as a system property key

(this is different to `Config` and `ConfigProvider` where we use config keys without the `dd.` prefix - in that part of the code we automatically add the `dd.` prefix when looking up system properties and `DD_` when looking up environment variables, but here we're going direct to `System.setProperty` so should include the prefix.)